### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bjornj12**
Creating Issue templates for bug reports and feature requests

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
```
Comment (hugihlynsson) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Does this field make sense as GitHub has a native assignee feature?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+** Using Warehouse **
+- [ ] Snowflake
+- [ ] BigQuery
+- [ ] Postgres
+- [ ] Redshift
+- [ ] Other....
```
Comment (hugihlynsson) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Nice 👌

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# avo-audit v1.0.0
+
```
Comment (hugihlynsson) [inline CHANGELOG.md:2]: Should we have an `Unreleased` section at the top of the changelog that users can add to before it get's batched up in a release?

**Review by hugihlynsson — APPROVED** [review]
Some suggestions but still 👌

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (bjornj12) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: good point, labels as well. This is something that came from github template

File: CHANGELOG.md (hunk 3 above)
Comment (bjornj12) [reply CHANGELOG.md:8]: Sounds good, added it on top

**Pull request merged**